protocols/gossipsub/src/handler.rs

```diff
@@ -469,20 +406,19 @@ impl ConnectionHandler for Handler {
                                     self.outbound_substream =
                                         Some(OutboundSubstreamState::PendingFlush(substream))
                                 }
-                                Err(HandlerError::MaxTransmissionSize) => {
-                                    error!("Message exceeded the maximum transmission size and was not sent.");
-                                    self.outbound_substream =
-                                        Some(OutboundSubstreamState::WaitingOutput(substream));
-                                }
                                 Err(e) => {
-                                    error!("Error sending message: {}", e);
-                                    return Poll::Ready(ConnectionHandlerEvent::Close(e));
+                                    log::debug!(
```

Comment:
The logs for errors on outbound substreams have all been downgraded to `debug` while the inbound ones are kept as `warn`. I would expect the `pub` part of gossipsub to be as important (if not more) as the `sub` part to keep these all as `warn` at least

Reply:
To me, a `warn` always means "Should I wake an ops person at 3am because of this"? Connections can die at any time because somebody e.g. closes their laptop. That is not a reason to wake an ops person IMO, hence I am gonna downgrade the inbound streams to debug instead upgrading the outbound ones to warn.